main.py

```diff
@@ -175,5 +173,48 @@ async def main():
     )
 
 
-if __name__ == "__main__":
+async def query(args):
+    con = sqlite3.connect("db.sqlite")
+    con.row_factory = sqlite3.Row
+    with con:
+        cur = con.cursor()
+        cur.execute(
+            """SELECT domain, software_name, software_version, strftime('%Y-%m-%d', datetime(last_updated_at, 'unixepoch')) FROM instances WHERE software_name = 'pixelfed'"""
+        )
```

Comment:
You probably want to parametize this, so:

```python
cur.execute(
    """SELECT domain, software_name, software_version, strftime('%Y-%m-%d', datetime(last_updated_at, 'unixepoch')) FROM instances WHERE software_name = ?""",
    (args.software,)
)
```

Or something.

Reply:
Ooooh, good idea!

Reply:
It'll help that https://github.com/bailey-coding/fediverse-peer-explorer/pull/7 is actually starting to store non-pixelfed looking software and instead all domains + software now